websites/docs-smoke-test/src/content/views/nested-headings.mdx

```diff
@@ -109,3 +109,41 @@ Content.
 ## L2 Long Subsection 4.2 - Lorem ipsum dolor sit amet, consectetur adipiscing elit
 
 Content.
+
+# Content Menu With First Level Heading
+
+<ChildSectionsNav parent="content-menu-with-first-level-heading"/>
+
+#### Lorem
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur scelerisque metus nec nibh faucibus, sed faucibus ipsum rutrum. Nullam nibh lectus, eleifend a nulla sit amet, pretium hendrerit purus. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Suspendisse sit amet eros ut felis iaculis pharetra at condimentum odio. Duis gravida orci non lacus dignissim, in laoreet ex ultrices. Cras consectetur dolor risus, nec porttitor velit vehicula volutpat. Pellentesque sit amet finibus nulla, ut egestas lacus.
+
+Cras pulvinar est eget nibh vestibulum, id ultrices orci auctor. Nullam auctor imperdiet ligula, ac auctor ligula aliquet sed. Suspendisse potenti. Vivamus tristique suscipit arcu, eu auctor libero venenatis sed. Maecenas ut cursus justo. Fusce euismod augue vitae pellentesque porttitor. Proin consectetur libero tempus nisi interdum auctor.
```

Comment:
![image](https://user-images.githubusercontent.com/4006139/92154245-c8e6f000-ee25-11ea-8a7a-576edfb3f494.png)

This scenario is invalid content and does not need to be "tested" (it doesn't actually test something because the index nav isn't shown anyways in that scenario).   Skipping a heading level is only allowed level two downwards at the moment.